WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1086,6 +1091,30 @@ import WordPressComAnalytics
         }
     }
 
+    func backgroundFetch(_ completionHandler: ((UIBackgroundFetchResult) -> Void)?) {
```

Comment:
I was following what we were doing on PushNotificationsManger, but in this case the correct thing to do is to have it as non-optional and @escaping, will update accordingly. Thx